frontend/src/metabase/new_query/containers/NewQueryOptions.jsx

```diff
@@ -28,7 +28,8 @@ const mapStateToProps = state => ({
 });
 
 const mapDispatchToProps = {
-  prefetchMetadata: () => Database.actions.fetchList({ include_tables: true }),
+  prefetchMetadata: () =>
```

Comment:
Have you verified if this actually avoids needing to fetch individuals dbs/schemas?

Reply:
Yup! I manually verified and added [a test](https://github.com/metabase/metabase/blob/c55bf11d9a74445450ab56a98ef04a2f2409e0c8/frontend/test/metabase/scenarios/question/query_builder_loading.cy.spec.js), which I'm "meh" on the utility of.

It doesn't avoid fetching the saved questions db, but I think that's better. We only do that big load if the user actually wants to see it.